cirq/ops/three_qubit_gates.py

```diff
@@ -161,14 +161,21 @@ class ThreeQubitDiagonalGate(gate_features.ThreeQubitGate):
     def __init__(self,
                  diag_angles_radians: List[Union[float, sympy.Basic]]) -> None:
         """
+        A 3 qubit gate whose off-diagonal elements are zero and who on-diagonal
+        elements are all phases.
 
         Args:
             diag_angles_radians: The list of angles on the diagonal in radians.
-
+                If these values are $(x0, x1, \ldots , x^7)$ then the unitary
+                has on diagonal values $(e^{i x0}, e^{i x1}, \ldots, e^{i x_7$.
```

Comment:
nit: LaTex expression is not well-formed: curly brace and parenthesis are not closed. Also x0, x1 are not consistent with x_7 (either use underscore for all or none of the variables).

Reply:
fixed